config/settings.yml

```diff
@@ -5,7 +5,7 @@ support_email: becomingateacher@digital.education.gov.uk
 campaign_email: itt.engagement@education.gov.uk
 
 search_ui:
-  base_url: http://localhost:3002
+  base_url: https://find.localhost
```

Comment:
What's this?

Reply:
Can we get rid of it?

Reply:
The `view on find link` in dev didn't load the correct url

Reply:
It's still used